ui/src/pages/backoffice/Patron/PatronDetails/components/PatronActionMenu/PatronActionMenu.js

```diff
@@ -0,0 +1,29 @@
+import {
+  ScrollingMenu,
+  ScrollingMenuItem,
+} from '@pages/backoffice/components/buttons/ScrollingMenu';
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+export default class PatronActionMenu extends Component {
+  render() {
+    return (
+      <div className={'bo-action-menu'}>
+        <ScrollingMenu offset={this.props.offset}>
+          <ScrollingMenuItem label="Checkout" elementId="patron-checkout" />
+          <ScrollingMenuItem label="Current loans" elementId="current-loans" />
+          <ScrollingMenuItem label="Loan requests" elementId="loan-requests" />
+          <ScrollingMenuItem
+            label="Literature requests"
+            elementId="literature-requests"
+          />
+          <ScrollingMenuItem label="Loans history" elementId="loans-history" />
+        </ScrollingMenu>
+      </div>
+    );
+  }
+}
+
+PatronActionMenu.propTypes = {
+  offset: PropTypes.number.isRequired,
```

Comment:
Can we provide a sensible default instead of making the offset required?

Reply:
I wanted to enforce the attention in this way, so the value is really set by a developer and the menu is not just flying around in case of changes

Reply:
@kprzerwa you know better!

Reply:
The problem I have noticed with the defaults it is that sometime we have misspelled the props name and we don't realize the error. So, use with cautions!